nix/sources.json

```diff
@@ -29,10 +29,10 @@
         "homepage": "",
         "owner": "DavHau",
         "repo": "pypi-deps-db",
-        "rev": "856d67ab093a68425c3896ec2961cea3b95ae93f",
-        "sha256": "0a7avm4xvm454gxy4dq0fc19j3f9ik8gf3kjpsqhszfkamrn9y0p",
+        "rev": "a44853469d0d6bc65cd382f24e4e60ff59bd60b2",
```

Comment:
The required version of collections-extended is very new and not present in the pypi-deps-db version we were using before.